payments-core/AndroidManifest.xml

```diff
@@ -22,6 +22,7 @@
             android:name=".view.PaymentRelayActivity"
             android:theme="@style/StripeTransparentTheme"
             android:exported="false" />
+        <activity android:name=".view.TestActivity"/>
```

Comment:
We don't want this here. It should be in a test only source set.

Reply:
my bad, fixed